lib/modules/platform/bitbucket-server/index.spec.ts

```diff
@@ -534,7 +534,7 @@ describe('modules/platform/bitbucket-server/index', () => {
             .reply(200, {
               mergeConfig: null,
             });
-          const actual = await bitbucket.getRepoForceRebase();
+          const actual = await bitbucket.getBranchForceRebase?.('main');
```

Comment:
```suggestion
          const actual = await bitbucket.getBranchForceRebase('main');
```
isn't it there? or use this if it's optional on type, because we know it must exist here.

```suggestion
          const actual = await bitbucket.getBranchForceRebase!('main');
```